packages/web-twig/src/Resources/components/TextArea/stories/TextAreaInline.twig

```diff
@@ -1,9 +1,9 @@
-<div style="display: flex; gap: 1rem; align-items: flex-end">
+<div style="display: flex; gap: 1rem; align-items: flex-start">
```

Comment:
There is a different alignment in the web package:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/2c752844-54bf-40f5-a207-77d438e2ed04)